app/views/admin/sign_ins/show.html.erb

```diff
@@ -8,10 +8,13 @@
         '/auth/github',
         class: 'usa-button usa-button usa-button-outline' %>
     </p>
-    <div class="mb1 right-align login_gov">
-      <%= link_to 'Authorize with Login.gov',
-        '/auth/saml',
-        class: 'usa-button usa-button usa-button-outline' %>
-    </div>
+    <p>
+      <a href='/auth/saml' class='usa-button usa-button usa-button-outline'>
```

Comment:
can we make this a `link_to` rather than href? that would be more railsy. Just change what you have to

```
<%= link_to '/auth/saml`, class: 'usa-button usa-button usa-button-outline' do %>
```
and then include the image tag within.